content/blog/2020/08/2020-08-15-external-fingerprint-storage-phase-3.adoc

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: "External Fingerprint Storage Phase-3 Updates"
```

Comment:
Would it make sense to rename it to Introducing the PostgreSQL Fingerprint Storage Plugin" or so?"

Reply:
You can have it both ways: 
`"External Fingerprint Storage Phase-3 Update: Introducing the PostgreSQL Fingerprint Storage Plugin"`